src/modules/api/pipeline-task/routes/task.py

```diff
@@ -37,43 +37,29 @@ def start_task(task_route):
   if op_id is None:
     logger.error(f'Request body must define an operation ID. Payload: {payload}')
     raise APIUnprocessableEntity('Request body must define an operation ID')
-  else:
-    call_id = f'TASK {op_id}'
-    logger.info(f'[{ call_id }] Payload: {payload}')
 
+  # Log the start of the task
+  call_id = f'TASK {op_id}'
+  logger.info(f'[{ call_id }] Starting job in queue { task_route }. Payload: {payload}')
 
-  # Create & start the job
-  try:
-    handler, create_response, run_response = start_job(payload, task_route, run_if_exists=True)
-    update_status_safe(queue, op_id, status=TaskStatus.RUNNING)
-
-  # Intercept API errors to add task ID
-  except APIError as ex:
-    update_status_safe(queue, op_id, status=TaskStatus.ERROR)
-    ex.set_call_id(call_id)
-    raise ex
-
-  # Wrap generic exceptions in an Internal Error class
-  except Exception as ex:
-    update_status_safe(queue, op_id, status=TaskStatus.ERROR)
-    raise APIInternalError('Failed to create job', call_id) from ex
+  # Try to create a task handler of the appropriate type
+  handler = get_task_handler(task_route, **payload)
 
-
-  # Create a Pipeline Operation record for the task
```

Comment:
The diff is a bit misleading here.  Creating & updating the `PipelineOperation` execution record is now handled implicitly by the `GCPRunner` class and its subclasses.  The new code uses the `Runner` subclass's `run` method to start the job, instead of the old (removed) function `start_job`.